tools/windows-node-installer/pkg/cloudprovider/azure/vm.go

```diff
@@ -933,6 +933,37 @@ func (az *AzureProvider) destroyDisk(ctx context.Context, vmInfo compute.Virtual
 	return
 }
 
+// DestroyWindowsVMWithResources destroys the Windows VMs along with all the errors associated with it
+func (az *AzureProvider) DestroyWindowsVM(vmName string) error {
+	ctx := context.Background()
+	_, nicName := az.getNICname(ctx, vmName)
```

Comment:
Why are we dropping these errors

Reply:
Old pattern, I just copied it. It's interesting that the azure api guys have error as first argument and necessary return value as second argument. I don't mind adding it.